sass/base/components/card-block/_card.scss

```diff
@@ -52,7 +52,7 @@
       }
     }
   }
-  &[class*="bg--"]{
+  [class*="bg--"] & {
```

Comment:
if we have `<div class="pragraph bg--red"><div class="card bg--black>`?

Reply:
good point